Upstream: yes, a version of it is in master
Reason: glibc already defines __USE_GNU based on _GNU_SOURCE
--- a/platform.c	2023-07-06 10:36:16.000000000 +0000
+++ b/platform.c	2023-09-14 11:45:23.738170635 +0000
@@ -37,7 +37,9 @@
 #   include <syslog.h>
 #   include <signal.h>
 #   include <errno.h>
+#ifndef __USE_GNU
 #define __USE_GNU
+#endif
 #include <dlfcn.h>
 #endif
 
